cli/cli/commands/service/add/add.go

```diff
@@ -112,6 +112,7 @@ var (
 		portIdSpecDelimiter,
 		serviceAddSpec,
 	)
+	noExperimentalFeature []kurtosis_core_rpc_api_bindings.KurtosisFeatureFlag
```

Comment:
So with just 'add' I can't add the same service twice?

Makes sense I guess. It's just one instruction.

Reply:
Yes for now add will not use instruction caching. Once we remove the feature flag and it becomes broadly available, we will start doing caching here as well